schema/cargo.kdl

```diff
@@ -0,0 +1,172 @@
+@kdl:schema "https://github.com/kdl-org/kdl/blob/main/schema/kdl-schema.kdl"
+
+metadata {
+    // TODO: update this link when we're ready to release something.
+    link "https://github.com/kdl-org/kdl/blob/main/schema/cargo.kdl" rel=self
+    title "Cargo Schema" lang=en
+    description "KDL-based translation of the Cargo.toml schema." lang=en
+    author "Kat Marchán" {
+        link "https://github.com/zkat" rel=self
+    }
+    link "https://github.com/kdl-org/kdl" rel=documentation
+    link "https://doc.rust-lang.org/cargo/reference/manifest.html" rel=documentation
+    license "Creative Commons Attribution-ShareAlike 4.0 International License" spdx=CC-BY-SA-4.0 {
+        link "https://creativecommons.org/licenses/by-sa/4.0/" lang=en
```

Comment:
I think this is a really good use of `lang`! It's possible to imagine software that has a link for license or documentation but the exact URL depends on the language your computer is configured to use (if the website doesn't automatically reroute you)!